src/main/java/org/ow2/proactive/connector/iaas/cloud/CloudManager.java

```diff
@@ -53,7 +70,14 @@ public CloudManager(List<CloudProvider> cloudProviders) {
     }
 
     public Set<Instance> createInstance(Infrastructure infrastructure, Instance instance) {
-        return cloudProviderPerType.get(infrastructure.getType()).createInstance(infrastructure, instance);
+        return cloudProviderPerType.get(infrastructure.getType()).createInstance(infrastructure,
+                                                                                 instance,
+                                                                                 Tag.builder()
+                                                                                    .key(Optional.ofNullable(connectorIaasTagKey)
```

Comment:
Removed with new TagManager component in cc604019f65fd0c272bddf7fe5578cd7dcff57aa